src/caret_analyze/record/records_service/response_time.py

```diff
@@ -1014,242 +911,3 @@ def _create_worst_to_best_case_record(
         for column in columns[1:]:
             record_dict[column] = record.get(column)
         return RecordFactory.create_instance(record_dict)
-
-
-class ResponseTimeseries:
-
-    def __init__(
-        self,
-        response_records: ResponseRecords
-    ) -> None:
-        self._records = response_records
-
-    def to_best_case_timeseries(self):
-        records = self._records.to_range_records()
-        input_column = records.columns[1]
-        output_column = records.columns[-1]
-        return self._to_timeseries(input_column, output_column)
-
-    def to_worst_case_timeseries(self):
-        records = self._records.to_range_records()
-        input_column = records.columns[0]
-        output_column = records.columns[-1]
-        return self._to_timeseries(input_column, output_column)
-
-    def _to_timeseries(self, input_column, output_column):
-        records = self._records.to_range_records()
-
-        t_ = records.get_column_series(input_column)
-        t_in = np.array(t_, dtype=np.int64)
-
-        t_out_ = records.get_column_series(output_column)
-        t_out = np.array(t_out_, dtype=np.int64)
-
-        latency = t_out - t_in
-
-        return t_in, latency
-
-    def to_best_case_records(self) -> RecordsInterface:
```

Comment:
In the ResponseTime class, shouldn't the following section be deleted as it seems to use the erased to_best_case_records?
` def to_best_case_records(self) -> RecordsInterface:`